runtime/compiler/env/j9method.cpp

```diff
@@ -3365,6 +3368,7 @@ void TR_ResolvedJ9Method::construct()
       {x(TR::java_lang_J9VMInternals_getSuperclass,                       "getSuperclass", "(Ljava/lang/Class;)Ljava/lang/Class;")},
       {x(TR::java_lang_J9VMInternals_identityHashCode,                    "identityHashCode", "(Ljava/lang/Object;)I")},
       {x(TR::java_lang_J9VMInternals_fastIdentityHashCode,                "fastIdentityHashCode", "(Ljava/lang/Object;)I")},
+      {x(TR::java_lang_J9VMInternals_valueHashCode,                       "valueHashCode", "(Ljava/lang/Object;)I")},
```

Comment:
I don't see `valueHashCode` being processed in this PR. Is this addition here to pave the way for future enhancement?

Reply:
I started off expecting that we almost certainly would want to have the JIT compiler recognize `valueHashCode`, so that it could generate inline code to calculate the default hash code for value types in simple cases.  But then it looked like, from performance measurements that you shared off-line, that that wouldn't be necessary.  I will remove this line.

Reply:
I've removed this in commit 516a1ed7f5898f989cff52d4806a93520529fcdb.  I can squash that down once reviews have been completed.